### PR TITLE
Proofreading3

### PR DESCRIPTION
**Pull request by pletzer**
Hi @chrisdjscott.
I reorganized the slurm training. Extended the list of commands that are useful with sinfo and sacct, the latter is used extensively at the end. We need to do a better job at:
1. describing the slurm script syntax
2. the srun command
3. how to submit with sbatch
4. how to check progress

### CODE REVIEW

File: _lessons/maui-and-mahuika/07-installing-local-packages.md [hunk 1]
```diff
@@ -20,29 +20,22 @@ Make sure to load one of the Python versions installed on Mahuika, e.g.
 ```
 module load Python/2.7.14-gimkl-2017a
 ```
-as this will provide you with many Python modules. You can check if a module is 
-already installed by typing, for instance,
+as this will provide you with many Python modules. Our Python distributions come with many modules already installed. You can check if a Python module is installed by typing, for instance, 
 ```
-python -c "import pnumpy"
+python -c "import numpy"
```
Comment (chrisdjscott) [inline _lessons/maui-and-mahuika/07-installing-local-packages.md:25]: Why the change to `numpy` - I thought we were using `pnumpy` as an example?

Comment (pm67nz) [reply _lessons/maui-and-mahuika/07-installing-local-packages.md:25]: It seems pnumpy is still the example for installing yourself, so that's OK.  

I will be taking out any mention of the sbatch option --exclusive.
